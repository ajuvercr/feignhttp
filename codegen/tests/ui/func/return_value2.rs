use feignhttp_codegen::get;

// error: return value must be Result
//    |  pub async fn send_get() -> i32 {}
//    |      ^^^^^^^^^^^^^^^^^^^^^^^^^^

#[get("http://xxx")]
pub async fn send_get() -> i32 {}

fn main() {}
