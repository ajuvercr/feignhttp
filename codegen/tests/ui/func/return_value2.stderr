error: return value must be Result
 --> tests/ui/func/return_value2.rs:8:5
  |
8 | pub async fn send_get() -> i32 {}
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^
